server.babel.js

```diff
@@ -1,8 +1,9 @@
 //  enable runtime transpilation to use ES6/7 in node
 
 var fs = require('fs');
+var path = require('path');
 
-var babelrc = fs.readFileSync('./.babelrc');
+var babelrc = fs.readFileSync(path.resolve(__dirname, 'node_modules/universal-redux/.babelrc'));
```

Comment:
@andresgutgon while it doesn't feel great to explicitly reach into `node_modules`, it allows universal-redux projects to work without having a project level `.babelrc`. ideally this `server.babel.js` file wouldn't be used at all, as it's only used to allow es6 files to be transpiled on the fly.